server/app.ts

```diff
@@ -54,7 +54,7 @@ export async function main() {
 	await buildClients();
 	buildRateLimiter();
 
-	if (process.argv.includes("--validate")) {
+	if (process.argv.includes("--validate") || process.argv.includes("-v")) {
```

Comment:
Lets leave the short alias out for now. `-v` is usually short for `--version`, and I want to minimize breaking changes in the future.

```suggestion
	if (process.argv.includes("--validate")) {
```